app/enterprise/0.32-x/upgrades-migrations.md

```diff
@@ -3,28 +3,29 @@ title: Upgrades and Migrations
 ---
 
 # Upgrades and Migrations
-## Upgrades of Kong EE
+## Upgrades of Kong Enterprise Edition (EE)
 
-We define an upgrade, the process of changing the version of the software currently deployed to a more recent one, there is no difference with concept of update; normally associated with an upgrade between minor versions.
+An upgrade (or update) is the process of changing the version of the software currently deployed to a more recent one, there is no difference with concept of update. We suggest following the below steps to perform an effective upgrade; notice, however, that this approach will require downtime, in case you need high availability review the Blue/Green Deployment section below.
 
-We suggest to follow the below steps to perform an effective upgrade, this approach will require downtime, in case you need high availability review the Blue/Green Deployment section.
-
-1. Download the destination version of the product you are interest to upgrade to in Bintray
-2. Take a backup copy of the production database
-3. Review the [changelog](https://docs.konghq.com/enterprise/changelog/) for the destination version of the upgrade
-4. Prepare the preproduction environment with the copy of the database taken from production
+1. Download the version of Kong you are interested in upgrading to.
+2. Make a backup copy of the production database.
+3. Review the [changelog](https://docs.konghq.com/enterprise/changelog/) for the version you are upgrading to.
+4. Prepare the preproduction environment with the copy of the database taken from production.
 5. Install the newer package in preproduction.
```

Comment:
What does "newer package" refer to? Please be more specific - like "Install the new version of Kong..."

Reply:
"New version of Kong" is the implicit subject of the whole document. Not necessary, redundant.